.github/workflows/release.yml

```diff
@@ -201,8 +201,45 @@ jobs:
           name: builds-${{ matrix.platform }}
           path: builds/*
 
+  build_windows:
+    if: github.repository_owner == 'viamrobotics'
+    needs: [prepare]
+    runs-on: windows-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - target: aarch64-pc-windows-gnullvm
+            platform: windows_arm64
+          - target: x86_64-pc-windows-gnu
+            platform: windows_x86_64
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.prepare.outputs.sha }}
+      - name: Setup rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+          targets: ${{ matrix.target }}
+      - name: Setup build directory
+        run: mkdir builds
+      - name: Build
+        run: |
+          cargo build --release --target=${{ matrix.target }}
+      - name: Copy
+        run: |
+          cp target/${{ matrix.target }}/release/viam_rust_utils.dll builds/libviam_rust_utils-${{ matrix.platform }}.dll
+          cp target/${{ matrix.target }}/release/viam_rust_utils.lib builds/libviam_rust_utils-${{ matrix.platform }}.lib
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: builds-${{ matrix.platform }}
+          path: builds/*
```

Comment:
Made reasonable modifications based on research/testing locally, but this is all untested as a CI workflow; didn't want to mess around with trying to make actual builds when running from a PR on my fork, and I figure it makes some sense to push the release stuff specifics to a separate PR. Happy to discuss if there's any disagreement or concern.

Reply:
Curious to see if the cross-compile works between aarch64 and x86 on windows

Reply:
We'll find out but I suspect it will, given that we're setting up the rust toolchain for the specific architecture and `cargo` supports cross-compilation out of the box.